embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/CustomerLogModule.kt

```diff
@@ -63,4 +69,23 @@ internal class CustomerLogModuleImpl(
             workerThreadModule.backgroundWorker(WorkerName.REMOTE_LOGGING)
         )
     }
+
+    override val logService: LogService by singleton {
+        EmbraceLogService(
+            openTelemetryModule.logWriter,
+            initModule.clock,
+            essentialServiceModule.metadataService,
+            essentialServiceModule.sessionIdTracker,
+            workerThreadModule.backgroundWorker(WorkerName.OTEL_LOGGING)
```

Comment:
Generally speaking we want to avoid creating background threads unless it's absolutely essential. Could we reuse the existing remote logger worker for processing logs? Does the `LogOrchestrator` need its own dedicated worker?

Reply:
Switched to using the existing log thread.